src/main/scala/org/allenai/ike/IkeToolWebapp.scala

```diff
@@ -98,18 +94,14 @@ class IkeToolActor extends Actor with HttpService with SprayJsonSupport with Log
 
   val serviceRoute = pathPrefix("api") {
     parameters('corpora.?) { corpora =>
+      val readySearchAppsWithDefaultSelected =
+        searchApps.filter(app => app._2.isCompleted && app._2.get.defaultSelected)
```

Comment:
If you want, you could use pattern matching and write `searchApps.filter { case (_, app) => app.isCompleted && app.defaultSelected }`, but this is fine, too.